package.json

```diff
@@ -71,17 +75,20 @@
     "js-yaml": "^3.13.1",
     "json-stable-stringify-without-jsonify": "^1.0.1",
     "levn": "^0.4.1",
-    "lodash": "^4.17.21",
+    "map-values": "^1.0.1",
     "minimatch": "^3.0.4",
     "natural-compare": "^1.4.0",
+    "omit": "^1.0.1",
     "optionator": "^0.9.1",
     "progress": "^2.0.0",
     "regexpp": "^3.1.0",
+    "rfdc": "^1.3.0",
     "semver": "^7.2.1",
     "strip-ansi": "^6.0.0",
     "strip-json-comments": "^3.1.0",
-    "table": "^6.0.4",
+    "table": "^6.0.9",
```

Comment:
This change should be dropped since `table` will be dropped by https://github.com/eslint/eslint/pull/14316

Reply:
It's fine to have this change included since this PR can be merged for a minor or patch release, while https://github.com/eslint/eslint/pull/14316 has to wait for the next major.